app/controllers/events_controller.rb

```diff
@@ -10,7 +10,7 @@ def create
   end
 
   def index
-    @events = Event.order('start_date desc')
+    @events = Event.public_events.order('start_date desc')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.